pipeline/lab.py

```diff
@@ -54,10 +68,10 @@ class ModifiedGene(dj.Manual):
 @schema
 class Subject(dj.Manual):
     definition = """
-    subject_id          : int   # institution 6 digit animal ID
+    subject_id          : varchar(10)
```

Comment:
is our int assumption just wrong initially, or is this tracking a different identifier and we need to find some way to integrate vs map?